### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Barqawiz**
add reference to Intelli, a framework to automate ML models integration as a graph.

### CODE REVIEW

**Review by strickvl — CHANGES_REQUESTED** [review]
Thanks @Barqawiz. Could you make sure that it's alphabetically ordered?

**Comment by Barqawiz** [comment]
Done alphabetical order fixed.

File: README.md [hunk 1]
```diff
@@ -443,6 +443,7 @@ Contributions are most welcome, please adhere to the [contribution guidelines](c
 - [Hypernets](https://github.com/DataCanvasIO/Hypernets) ![](https://img.shields.io/github/stars/DataCanvasIO/Hypernets.svg?style=social) - A General Automated Machine Learning Framework.
 - [Hyperopt](https://github.com/hyperopt/hyperopt) ![](https://img.shields.io/github/stars/hyperopt/hyperopt.svg?style=social) - Distributed Asynchronous Hyperparameter Optimization in Python.
 - [hyperunity](https://github.com/gdikov/hypertunity) ![](https://img.shields.io/github/stars/gdikov/hypertunity.svg?style=social) - A toolset for black-box hyperparameter optimisation.
+- [Intelli](https://github.com/intelligentnode/Intelli) ![](https://img.shields.io/github/stars/intelligentnode/Intelli?style=social) - a framework to connect a flow of ML models by applying graph theory.
```
Comment (strickvl) [inline README.md:446]:
```suggestion
- [Intelli](https://github.com/intelligentnode/Intelli) ![](https://img.shields.io/github/stars/intelligentnode/Intelli?style=social) - A framework to connect a flow of ML models by applying graph theory.
```

**Review by strickvl — APPROVED** [review]
Thanks!

**Pull request merged**